pyrasterframes/src/main/python/docs/__init__.py

```diff
@@ -40,6 +40,31 @@ def resource_dir_uri():
     return 'file://' + resource_dir()
 
 
+def folium_map(vector_data):
```

Comment:
@vpipkt The one problem with this is that I'd like to get rid of `docs/__init__.py` So that the notebook versions can be self-contained. I can't remember the original reason this is required (as I thought `folium` has `display` methods)... Is it just for the `html` version?

Reply:
good thought there.

Reply:
So the formatter for `text/markdown` was the key but it seems that needed to be done outside `docs/__init__.py` script.

Reply:
@vpipkt 

> So the formatter for text/markdown was the key but it seems that needed to be done outside docs/__init__.py script.

Not sure I know what you mean... Do you have reservations about `pyrasterframes/rf_ipython.py`?